geolocator_platform_interface/pubspec.yaml

```
@@ -3,7 +3,7 @@ description: A common platform interface for the geolocator plugin.
 homepage: https://github.com/baseflow/flutter-geolocator
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.3.4
+version: 2.4.0
```

Update the patch number:
```suggestion
version: 2.3.5
```